project/Dependencies.scala

```diff
@@ -75,6 +75,28 @@ object Dependencies {
     val test            = Seq(scalaCheck)
   }
 
+  object Typelevel {
+    val mouse                  = "org.typelevel" %% "mouse"                    % "1.3.2"
+  }
```

Comment:
Since it's only one lib entry anyway...

```suggestion
    val mouse                  = "org.typelevel" %% "mouse"                    % "1.3.2"
```

And maybe move "`mouse`" under `monix`.

Reply:
addressed as part of [4f9b79f9f8c7eb8](https://github.com/sky-uk/kafka-message-scheduler/pull/367/commits/4f9b79f9f8c7eb8ac990fd5c640dac8954d7c0e3)